README.md

```diff
@@ -157,7 +157,7 @@ from pydantic import BaseModel, constr
 
 # Define a Feature model with Geometry as `Point` and Properties as a constrained Model
 class MyProps(BaseModel):
-    name: constr(regex=r'^(drew|vincent)$')
+    name: constr(pattern=r'^(drew|vincent)$')
```

Comment:
Personally I prefer `name: Annotated[str, Field(pattern=r"^(drew|vincent)$")`. As some type checking didn't like using `constr` as a type. But may also depend on python versions too. So probably doesn't make a ton of difference in the readme.